package.json

```diff
@@ -15,7 +15,7 @@
     "migrations": "cli.js"
   },
   "scripts": {
-    "test": "jest && npm run lint",
+    "test": "jest --coverage && npm run lint",
```

Comment:
Wenn wir coverage an machen, dann müssen wir entweder noch zusätzliche tests schreiben, oder die coverage thresholds hier in der jest configuration anpassen.
Aktuell sind das sinnvolle Werte, die aber ja leider nicht erreicht werden.

Reply:
ich habs erstmal rausgenommen... wenn wir was sinnvolles haben können wir die ja wieder mit rein nehmen